### PR TITLE
Expand/collapse tables for find-in-page feature in PageFragment

### PR DESCRIPTION
**Pull request by cooltey**
https://phabricator.wikimedia.org/T130403

### CODE REVIEW

**Comment by cooltey** [comment]
Created another patch for fully expanding/collapsing tables, now this patch is waiting for it to be on production.

**Comment by cooltey** [comment]
The PR is ready for review. Please note that the `mobile-html` should be fully refreshed and you may need to clear the cache/data if you are not able to see the tables are expanded when tapping on "find-in-page" button.

File: app/src/main/java/org/wikipedia/page/PageFragment.java [hunk 1]
```diff
@@ -919,43 +919,50 @@ void showFindInPage() {
...
+                public void onDestroyActionMode(ActionMode mode) {
+                    if (webView == null || !isAdded()) {
+                        return;
+                    }
+                    funnel.setPageHeight(webView.getContentHeight());
+                    funnel.logDone();
+                    webView.clearMatches();
+                    hideSoftKeyboard();
+                    setToolbarElevationEnabled(true);
+                    bridge.execute(JavaScriptActionHandler.expandCollapsedTables(false));
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/page/PageFragment.java:963]: I'm not sure we need to re-close the tables when this is done.
Suppose I open the find-in-page box, and it finds the text I'm looking for _inside_ a table. Then I exit out of find-in-page, but the table suddenly closes.

**Review by dbrant — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/wikipedia/page/PageFragment.java (hunk 1 above)
Comment (cooltey) [reply app/src/main/java/org/wikipedia/page/PageFragment.java]: Makes sense to me. Removed it.

**Pull request merged**